subsys/zigbee/cli/zigbee_cli_cmd_attr.c

```diff
@@ -18,7 +18,7 @@
 #define ATTRIBUTE_ENTRY_TIMEOUT_S  10
 
 
-LOG_MODULE_REGISTER(attr, CONFIG_ZIGBEE_SHELL_LOG_LEVEL);
+LOG_MODULE_DECLARE(zigbee_shell, CONFIG_ZIGBEE_SHELL_LOG_LEVEL);
```

Comment:
In nRF5 SDK the idea was to use logger subgroups for selecting logs from selected CLI module, thus we had `cli.attr` apart from the global `cli`. Do you see a value/chance to implement it that way in NCS?